### PR TITLE
View tint check change

### PR DESCRIPTION
**Pull request by jfbilodeau**
Minor optimization: Reduced the need for invalidateColorTransform(). This method is called when the tint property is set, whether or not the value has changed. Now, the colorMul and colorAdd property calls sinvalidateColorTransform if the value has changed.

### CODE REVIEW

**Comment by soywiz** [comment]
Hey @jfbilodeau can you make this PR in this repo: https://github.com/korlibs/korge-next ? All the new work happens there, and then I'm synchronizing all the changes here later

File: korge/src/commonMain/kotlin/com/soywiz/korge/view/View.kt [hunk 1]
```diff
@@ -372,7 +374,8 @@ abstract class View internal constructor(
     var colorAdd: ColorAdd
         get() = _colorTransform.colorAdd;
         set(v) {
-            _colorTransform.colorAdd = v
+            if (v.rgba != _colorTransform.colorAdd.rgba)
+                _colorTransform.colorAdd = v
             invalidateColorTransform()
```
Comment (soywiz) [inline korge/src/commonMain/kotlin/com/soywiz/korge/view/View.kt:379]: Can I asume you forgot `{ }` to include the invalidateColorTransform as part of the if?

**Comment by jfbilodeau** [comment]
Compile issues--will reopen when solved...

**Comment by soywiz** [comment]
But please, reopen it in `korge-next` plz. If not later it is pretty hard to me to synchronize all the stuff without making mistakes and removing already fixed stuff

**Comment by jfbilodeau** [comment]
Will do!

**Pull request closed without merging**